bin-src/lib/writeChromaticReport.js

```diff
@@ -0,0 +1,18 @@
+import jsonfile from 'jsonfile';
+import wroteReport from '../ui/messages/info/wroteReport';
+
+const { writeFile } = jsonfile;
+
+// Extract important information from ctx, sort it and output into a json file
+export async function writeChromaticReport(ctx) {
+  const chromaticReportPath = 'chromatic-report.json';
+  const { argv, client, env, help, http, log, pkg, task, title, ...restCtx } = ctx;
+  const reportContext = Object.keys(restCtx)
+    .sort((a, b) => a.localeCompare(b))
+    .reduce((acc, key) => ({ ...acc, [key]: restCtx[key] }), {});
+  return writeFile(chromaticReportPath, reportContext, { spaces: 2 })
+    .then(() => ctx.log.info(wroteReport(chromaticReportPath, 'Chromatic')))
```

Comment:
why use chain-style promises if it's in an `async` function?

Reply:
To catch errors without using try catch directly

Reply:
What does everyone hate `try / catch` ? ;)

Reply:
I don't hate it, just didn't use there. I can change if you'd like!

Reply:
Personally, I prefer to not mix async + promise code as I find it confusing. I'm not sure we have a strict rule about it though (perhaps we should? IDK).

Reply:
Alright changed! Let me know if there is anything else!